src/reactapp/src/pages/EvaluationForm.js

```diff
@@ -346,11 +346,39 @@ export default function EvaluationForm() {
     }
     return [{
       "label": "N/A - Not Assessed",
-      "value": "-1"
+      "value": -1
     }];
   }
 
 
+  /**
+   *
+   * Returns true if all the required scores are selected.
+   * @returns
+   */
+  const areAllScoreSelected = () => {
+    let hasPendingScores = false;
```

Comment:
@jleiva-gap 'Not Assessed' is a valid selection for the evaluator to choose, but if the evaluator wants to choose 'Not Assessed' they need to choose it, not have it defaulted. I think the best solution here is to have a blank choice and that becomes the -1/invalid choice.